salt/state.py

```diff
@@ -1903,6 +1903,29 @@ def apply_exclude(self, high):
                 high.pop(id_)
         return high
 
+    @staticmethod
+    def _add_to_extend(extend, name, state, rkey, rstate, id_):
+        found = False
+        if name not in extend:
+            extend[name] = HashableOrderedDict()
+        if rstate not in extend[name]:
+            extend[name][rstate] = []
```

Comment:
This should be using `state` instead of `rstate`. Naming the variables more verbosely will help some with following this.